lib/mockbuilder.py

```diff
@@ -195,12 +203,32 @@ def copy_results(self, package):
             self.timestamp, package.name)
         self._copy_rpms(package.build_cache_dir, package_build_results_dir)
 
+    def create_repository(self):
+        """
+        Create yum repository in build results directory.
+        """
+        result_dir = CONF.get('common').get('result_dir')
+        build_results_dir = os.path.join(
+            result_dir, 'packages', self.timestamp)
+        utils.run_command("createrepo %s" % build_results_dir)
+
+        repo_files_dir = os.path.join(result_dir, "repository_files")
```

Comment:
Perhaps the timestamp should be the father of the resulting descending directories of a build, for example:

```
result/<timestamp>/packages/<file>.rpm
result/<timestamp>/repository/<file>.repo
result/<timestamp>/repository/repodata/<file>.xml
result/<timestamp>/iso/<file>.iso
```